i18n/.js/src/test/scala/io/udash/i18n/BindingsTest.scala

```diff
@@ -158,22 +160,26 @@ class BindingsTest extends AsyncUdashFrontendTest {
           translatedDynamic(key2)(key => key(3, 3.14)),
           translatedDynamic(key3)(key => key(0.99)),
           translatedDynamic(key4)(key => key("test", 1, true, 3.1415)),
-          translatedDynamic(keyX)(key => key("test", 1.asInstanceOf[Any], true, 3.1415, "Udash"))
+          translatedDynamic(keyX, rawHtml = true)(key => key("test", 1.asInstanceOf[Any], true, 3.1415, "Udash"))
```

Comment:
How about a few more test cases for the raw binding? At least the success with 0 children case is tricky.

Reply:
@ddworak good idea. `key0` makes this trick.